src/browser/commands/types.ts

```diff
@@ -85,7 +85,7 @@ export interface AssertViewOpts extends Partial<AssertViewOptsConfig> {
 export type AssertViewCommand = (
     this: WebdriverIO.Browser,
     state: string,
-    selectors: string | string[],
+    selectors?: string | string[],
```

Comment:
I doubt this will allow doing `assertView('some-state', null, {...opts})`. null needs to be explicitly stated here.